.github/workflows/main.yml

```diff
@@ -20,12 +20,16 @@ jobs:
   build:
     needs: lint
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python: [3.7, 3.9]
```

Comment:
Is it not possible to use `latest` for a matrix?

Reply:
AFAIK it only accepts numerical values but giving a try with `latest`

Reply:
@epicadk it isn't possible , will revert the changes